spec/system/reports/export_data_spec.rb

```diff
@@ -62,6 +62,7 @@
     click_button "Mileage Report"
     wait_for_download
 
+    expect(download_file_name).to match(/mileage-report-\d{4}-\d{2}-\d{2}.csv/)
```

Comment:
This would be a better test if it froze time and asserted exact match on the frozen time string, but testing is good :)